integration_tests/test_client_id.py

```diff
@@ -0,0 +1,43 @@
+import json
+from pathlib import Path
+
+import pytest
+from pystarport import ports
+
+from .network import setup_custom_cronos
+from .utils import wait_for_port
+
+
+@pytest.fixture(scope="module")
+def custom_cronos(tmp_path_factory):
+    path = tmp_path_factory.mktemp("cronos")
+    yield from setup_custom_cronos(
+        path,
+        26800,
+        Path(__file__).parent / "configs/default.jsonnet",
```

Comment:
can this test case run on the default cronos, it'll be faster, without setup another devnet.